symbology/create-symbols-from-web-styles/src/main/java/com/esri/samples/create_symbols_from_web_styles/CreateSymbolsFromWebStylesSample.java

```diff
@@ -0,0 +1,316 @@
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Create Symbols From Web Styles Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the light gray basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_LIGHT_GRAY);
+
+      // set an initial reference scale on the map for controlling symbol size
+      map.setReferenceScale(100000);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(34.28301, -118.44186, 7000));
+
+      // create a feature layer from a service
+      FeatureLayer featureLayer = new FeatureLayer(new ServiceFeatureTable(
+        "http://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0"));
+
+      // add the feature layer to the map's operational layers
+      map.getOperationalLayers().add(featureLayer);
+
+      // display an error if the feature layer fails to load
+      featureLayer.addDoneLoadingListener(() -> {
+        if (featureLayer.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
+          new Alert(Alert.AlertType.ERROR, "Feature layer failed to load. Details: \n"
+            + featureLayer.getLoadError().getCause().getMessage()).show();
+        }
+      });
+
+      // create a unique value renderer
+      UniqueValueRenderer uniqueValueRenderer = new UniqueValueRenderer();
+
+      // add the name of a field from the feature layer data that symbols will be mapped to
+      uniqueValueRenderer.getFieldNames().add("cat2");
+
+      // set the unique value renderer on the feature layer
+      featureLayer.setRenderer(uniqueValueRenderer);
+
+      // create a symbol style from a web style
+      symbolStyle = new SymbolStyle("Esri2DPointSymbolsStyle", null);
+
+      // display an error if the symbol style fails to load
+      symbolStyle.addDoneLoadingListener(() -> {
+        if (symbolStyle.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
+          new Alert(Alert.AlertType.ERROR, "Error: could not load symbol style. Details: \n"
+            + symbolStyle.getLoadError().getMessage()).show();
+        }
+      });
+
+      // setup the UI for the legend
+      setupLegend();
+
+      // create a list of the required symbol names from the web style
+      ArrayList<String> symbolNames = new ArrayList<>();
+      symbolNames.addAll(Arrays.asList("atm", "beach", "campground", "city-hall", "hospital", "library", "park",
+        "place-of-worship", "police-station", "post-office", "school", "trail"));
```

Comment:
It's possible to instantiate the ArrayList and populate it with values in one statement:
```suggestion
      ArrayList<String> symbolNames = new ArrayList<>(Arrays.asList("atm", "beach", "campground", "city-hall", 
              "hospital", "library", "park", "place-of-worship", "police-station", "post-office", "school", "trail"));
```